src/evict/evict_inline.h

```diff
@@ -505,19 +505,22 @@ __wti_evict_updates_needed(WT_SESSION_IMPL *session, double *pct_fullp)
  *     thresholds.
  */
 static WT_INLINE bool
-__wt_evict_needed(WT_SESSION_IMPL *session, bool busy, bool readonly, double *pct_fullp)
+__wt_evict_needed(
```

Comment:
If app threads should not participate in eviction, what about __checkpoint_wait_reduce_dirty_cache? should this become a no-op, or is still checkpoint is expected to wait for dirty content to come down ?

Reply:
There is no checkpoint run on the standby.